core/src/com/unciv/logic/automation/city/ConstructionAutomation.kt

```diff
@@ -382,7 +392,22 @@ class ConstructionAutomation(val cityConstructions: CityConstructions) {
         if (foodBuilding != null) {
             var modifier = 1f
             if (city.population.population < 5) modifier = 1.3f
+            modifier *= city.civ.getPersonality().scaledFocus(PersonalityValue.Food)
             addChoice(relativeCostEffectiveness, foodBuilding.name, modifier)
         }
     }
+
+    private fun addFaithBuildingChoice() {
+        if (civInfo.gameInfo.isReligionEnabled()) return
+        val faithBuilding = statBuildings
+            .filter { it.isStatRelated(Stat.Faith) }
+            .filterBuildable()
+            .minByOrNull { it.cost }
+        if (faithBuilding != null) {
```

Comment:
`if (faithBuilding == null) return` for low nesting
Or just ?: return I'm fine either way

Reply:
`?:` is pretty ugly here

Reply:
Unless it's on the line above. Completely missed that, I guess Studio caught it for me